ultravox/model/data_processing.py

```diff
@@ -91,8 +110,33 @@ def _process(self, sample: datasets.VoiceSample) -> Dict[str, Any]:
             )["input_ids"].shape[-1]
             labels[:input_text_len] = -100
 
+            if self.include_alt_input:
+                alt_input_text = input_text.replace(
```

Comment:
Can this be simplified? The assistant message should have the same length for both input types, so can we simply do `alt_input_text_len = len(alt_text) - (len(text) - len(input_text))`?

Reply:
I think you are right, though the computation should be based on tokens, not text.

`alt_input_text_len = input_text_len + len(alt_input_ids) - len(input_ids)`

Reply:
ah yeah, good catch. maybe `alt_input_token_len` then to avoid future confusion.

Reply:
right, i will update `input_text_len` to `input_token_len` as well.